File: examples/simple/tcp_client.c
#include<sys/types.h>
#include<sys/socket.h>
#include<netdb.h>
#include<netinet/in.h>
#include<inttypes.h>
#include<arpa/inet.h>
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
#include<unistd.h>
#define PORTNUM 9000
//client
int main(void)
{
	int sd;
	char buf[256];
	struct sockaddr_in sin;
	
    while(1){
        if((sd=socket(AF_INET, SOCK_STREAM, 0)) == -1)
        {
            perror("socket");
            exit(1);
        }
        
        memset((char*)&sin, '\0', sizeof(sin));
        sin.sin_family = AF_INET;
        sin.sin_port = htons(PORTNUM);
        sin.sin_addr.s_addr = inet_addr("127.0.0.1");
        
        if(connect(sd, (struct sockaddr *)&sin, sizeof(sin)))
        {
            // perror("connect");
            // exit(1);
            sleep(0.1);
        }
        
        if(recv(sd,buf,sizeof(buf),0) == -1)
        {
            // perror("recv");
            // exit(1);
            sleep(0.1);
        }
        printf("From Server : %s\n", buf);
        close(sd);

        sleep(0.5);
        
    }
	

	return 0;
}

File: examples/simple/tcp_client.py
# import socket

# client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
# client_socket.connect(("127.0.0.1", 5000))
# while 1:
#     data = client_socket.recv(512).decode()
#     if ( data == 'q' or data == 'Q'):
#         client_socket.close()
#         break
#     else:
#         print ("RECEIVED:" , data)
#         data = input ( "SEND( TYPE q or Q to Quit):" )
#         if ( data == 'q' or data == 'Q'):
#             client_socket.send(data.encode())
#             client_socket.close()
#             break
#         else:
#             client_socket.send(data.encode())
# print ("socket colsed... END.")

import socket
import time


# sock.connect(('127.0.0.1', 9000))
# sock.send('hello'.encode())
while 1:
    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
    # sock.bind(('127.0.0.1', 9000))
    try:
        sock.connect(('127.0.0.1', 9000))
        data = sock.recv(256)
        print(data.decode("utf-8"))
    except:
        # print('fail to connect socket')
        pass
    sock.close()
    # time.sleep(0.1)